pkg/storage/ec/client.go

```diff
@@ -343,10 +343,15 @@ func (ec *ecClient) Get(ctx context.Context, limits []*pb.AddressedOrderLimit, p
 
 	rr, err = eestream.Decode(rrs, es, ec.memoryLimit, ec.forceErrorDetection)
 	if err != nil {
-		return nil, err
+		return nil, Error.Wrap(err)
+	}
+
+	ranger, err := eestream.Unpad(rr, int(paddedSize-size))
+	if err != nil {
```

Comment:
Nit: Error.Wrap returns nil if the input is nil.  I don't think you need this 'if' check.

Reply:
I do not like to abuse of it, but I think that's clear when nil is returned when no error than having to know that `Wrap` returns nil.

However I will change them